src/arcade/core/util/MiniBox.java

```diff
@@ -60,6 +66,31 @@ public int getInt(String id) {
         return (s == null || !s.matches(NUMBER_REGEX) ? 0 : Double.valueOf(s).intValue());
     }
 
+    /**
+     * Gets the value for given key converted to a distribution.
+     *
+     * @param id the key
+     * @param random the random number generator
+     * @return the distribution instance
+     */
+    public Distribution getDistribution(String id, MersenneTwisterFast random) {
```

Comment:
Just for my sanity - I think we handle case sensitivity elsewhere (InputLoader and InputParser), but these are not case sensitive, right?

Reply:
Added an explicit cast to uppercase for distribution types!